apps/docs/features/docs/Troubleshooting.ui.client.tsx

```diff
@@ -220,9 +241,9 @@ function TroubleshootingFilterInternal({
           <MultiSelector.Trigger badgeLimit={1} className="w-48" label="Error codes" />
           <MultiSelector.Content>
             <MultiSelector.List>
-              {errors?.map((error) => (
-                <MultiSelector.Item key={`error-${error.code}`} value={error.code}>
-                  {error.code}
+              {allErrorCodes?.map((error) => (
+                <MultiSelector.Item key={`error-${error}`} value={error}>
+                  {error}
```

Comment:
Still need to optional chain now that there's the `[]` fallback?

```suggestion
              {allErrorCodes.map((error) => (
                <MultiSelector.Item key={`error-${error}`} value={error}>
                  {error}
```

Reply:
good catch!